packages/vuetify/build/webpack.dev.config.js

```diff
@@ -85,30 +82,30 @@ module.exports = merge(baseWebpackConfig, {
     disableHostCheck: true
   },
   plugins: [
-    new VueLoaderPlugin(),
-    new ForkTsCheckerWebpackPlugin({
-      checkSyntacticErrors: true,
-      tsconfig: resolve('../tsconfig.json')
-    }),
-    new HappyPack({
-      id: 'ts',
-      threadPool: happyThreadPool,
-      loaders: [
-        'babel-loader',
-        {
-          loader: 'ts-loader',
-          options: {
-            appendTsSuffixTo: [/\.vue$/],
-            happyPackMode: true
-          }
-        },
-        'eslint-loader?cache=true?emitWarning=true'
-      ]
-    }),
-    new HappyPack({
-      id: 'js',
-      threadPool: happyThreadPool,
-      loaders: ['babel-loader', 'eslint-loader?cache=true?emitWarning=true']
-    })
+    new VueLoader.VueLoaderPlugin(),
+    // new ForkTsCheckerWebpackPlugin({
```

Comment:
?

Reply:
I couldn't get happypack to work.

Reply:
works fine here, just eslint-loader is broken because lib doesn't build properly any more

If you're going to disable it the loaders need to be copied from their respective happypack configs, we don't use babel-loader for typescript.